main/wallet-backend.js

```diff
@@ -49,18 +55,26 @@ class WalletBackend {
     } else {
       ({ seed, isNew } = await this.getSeedPhrase())
     }
-    this.provider = new Filecoin(new HDWalletProvider(seed), {
-      apiAddress: 'https://api.node.glif.io/rpc/v0'
-    })
-    this.address = await this.getAddress()
+    this.provider = new ethers.providers.JsonRpcProvider(
+      'https://api.node.glif.io/rpc/v0'
+    )
+    // this.provider.on('debug', d => console.log(JSON.stringify(d, null, 2)))
```

Comment:
do we wanna include this in the commit as a comment?

Reply:
Maybe we can use `debug` or `electron-log` instead of `console.log` and that way keep this line always in & uncommented?

Reply:
This is too noisy to keep in, I've removed it for now